source/utils.ts

```diff
@@ -24,3 +24,14 @@ export function getParserServices(
   }
   return context.parserServices;
 }
+
+export function getTypeCheckerAndNodeMap(context: Rule.RuleContext) {
```

Comment:
I added this function because I think these 3 lines will be repeated a lot in the future.